index.js

```diff
@@ -1,11 +1,24 @@
 const globals = require('globals');
 
+// Auto-detect plugin name, to avoid duplicate errors
+const testPlugin = (() => {
+    try {
+        require.resolve('jest');
+        return 'jest';
+    } catch {
+        return 'mocha';
+    }
+})();
+
 module.exports = {
     globals: {
         ...Object.fromEntries(Object.entries(globals.browser).map(([key]) => [key, 'off'])),
         ...globals.node,
-        ...globals.mocha,
     },
+    plugins: ['jest', 'mocha'],
+    extends: [
+        `plugin:${testPlugin}/recommended`,
```

Comment:
Maybe too strong, causes lots of errors...
→ just "mocha/no-exclusive-tests" and "jest/no-focused-tests" ?
→ or extend recomended plugins, then disable a few (no arrow...)?

Reply:
Maybe we can review on some repos what errors this produces and agree on the rules we want to disable? Hopefully not too many, as relaxed rules tend to cause heterogeneous code styles across our projects...

Some are quite obvious to disable, the others, if their number is ok, should be kept. We might as well generalize prettier, so it automates most of the work...